server/service/src/document/Programs.md

```diff
@@ -0,0 +1,144 @@
+# Programs module overview
+
+The program module supports doctors or practitioners to record medical patient data, to manage patients and to do patient related data reporting.
+A medical `program` focuses on the treatment of a medical condition and a patient can be enrolled in one or more programs.
+
+All program related data is recorded in `documents` which resemble physical forms.
+For example, there are currently three main document types:
+
+1. The `Patient` document records general patient information such as name, address or age.
+   The Patient document is shared between all programs of a patient.
+2. The `Program Enrolment` document holds patient specific information about a single program the patient is enrolled in.
+3. `Program Encounter` documents record data about a patients visit at a doctor.
+   Each program encounter is associate with a single program and there is an encounter document for every visit.
+
+What data is stored in the documents is widely customizable and its possible to add new custom document types.
+This makes the system customizable to fullfil the needs of different clients.
```

Comment:
```suggestion
Data stored in the documents are widely customizable and it is possible to add new custom document types.
This makes the system customizable to fullfil the needs of different clients.
```

Reply:
Did:
Data stored in the documents is widely customizable and it is possible to add new custom document types.